Userland/Libraries/LibCore/EventLoop.h

```diff
@@ -23,6 +23,10 @@
 #include <sys/time.h>
 #include <sys/types.h>
 
+#ifdef __serenity__
+extern bool s_global_initializers_ran;
+#endif
```

Comment:
Hmm, does everyone who includes `EventLoop.h` really need this? Seems like it could live in `EventLoop.cpp` instead :)

Also, this commit feels very un-atomic. Many different things happening here, can we split them apart?